index.js

```diff
@@ -63,11 +63,18 @@ client.on("message", (message) => {
     message.channel.send("Planning Poker finished");
     message.channel.send("Here is an overview of your game:");
 
-    for (const question of Poker.questions)
+    let totalStoryPoints = 0;
+    for (const question of Poker.questions) {
       message.channel.send(
         `Question: ${question.question} Story Points: ${question.storypoints}`
       );
+      totalStoryPoints += question.storypoints;
```

Comment:
`question.storypoints` is a String. They should be parsed to a number. Currently, when I play one question with 12 story points. It prints `Total Story Points: 012`

Reply:
This should actually be fixed with #25 I'm going to test it and may merge it